styles/elements/_headings.scss

```diff
@@ -39,8 +39,8 @@ h2,
 .h2,
 h3,
 .h3 {
-	font-family: $base-font-family-bold;
-	color: $dark-silk;
+  color: $dark-silk;
+  @include bold;
 }
```

Comment:
I think it's better to add the @include of mixin first and the color (and other properties) after that